libtonode-tests/tests/legacy.rs

```diff
@@ -4244,3 +4245,26 @@ mod basic_transactions {
 async fn proxy_server_worky() {
     zingo_testutils::check_proxy_server_works().await
 }
+
```

Comment:
I see this test will send dust too sapling from Orchard.   I'm not clear on correct behavior.